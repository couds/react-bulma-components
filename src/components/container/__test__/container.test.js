import React from 'react';
import renderer from 'react-test-renderer';
import Container from '..';

describe('Container component', () => {
  it('Should have container classname', () => {
    const component = renderer.create(
      <Container>
        <p className="bd-notification is-success">
          <p>Default</p>
          <p>Container</p>
        </p>
      </Container>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });

  it('Should have helper classnames', () => {
    const component = renderer.create(
      <Container
        paddingless
        responsive={{
          widescreen: {
            display: {
              value: 'block',
              only: true,
            },
          },
          touch: {
            display: {
              value: 'flex',
            },
          },
          desktop: {
            hide: {
              value: true,
            },
          },
        }}
      >
        <p className="bd-notification is-success">
          <p>Default</p>
          <p>Container</p>
        </p>
      </Container>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
