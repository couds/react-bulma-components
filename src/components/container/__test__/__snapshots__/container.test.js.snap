// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component Should have container classname 1`] = `
<div
  className="container"
  style={Object {}}
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;
